wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -314,6 +314,11 @@ private extension AppDelegate {
         // Get maxNumberAccounts form SecurityFlags or SessionManager.defaultMaxNumberAccounts if no MAX_NUMBER_ACCOUNTS flag defined
         let maxNumberAccounts = SecurityFlags.maxNumberAccounts.intValue ?? SessionManager.defaultMaxNumberAccounts
 
+        let analyticsSessionConfiguration = AnalyticsSessionConfiguration(
+            countlyKey: Bundle.countlyAppKey,
+            host: BackendEnvironment.shared.countlyURL
+        )
+
```

Comment:
I think it'd be better to do a guard let on the app key and url, and only if they're there created a configuration, and pass this optional configuration to the session manager.

Reply:
Done!